packages/ui/__stories__/SimpleTable.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+
+import { SimpleTable } from '../src'
+
+import styles from './SimpleTable.module.scss'
+
+export default {
```

Comment:
nit: can we also have a link to figma?

Reply:
We don't have this table in figma. This component replaces legacy table. In the future we could update its style @pawelklasa